src/Stratis.SmartContracts.Networks/NetworkNodes.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Stratis.Bitcoin;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.SmartContracts;
+using Stratis.Bitcoin.Features.SmartContracts.PoA;
+using Stratis.Bitcoin.Features.SmartContracts.PoS;
+using Stratis.Bitcoin.Features.SmartContracts.PoW;
+using Stratis.Bitcoin.Features.SmartContracts.Wallet;
+
+namespace Stratis.SmartContracts.Networks
+{
+    public static class NetworkNodes
+    {
+        public static IFullNodeBuilder GetPoWSmartContractNodeBuilder(NodeSettings settings)
+        {
+            return new FullNodeBuilder()
```

Comment:
The problem with this is that the Networks project will then need to have a reference to all these other projects.
Someone that would want to run a very lean node, or even that just wants to get the Network types, will have to download all these other dependencies.

Reply:
Yeah agreed, I think it shouldn't go in .Networks. 

The important part though is that we have a consistent build ordering method _somewhere_ to be consumed by both tests and daemons.

Any ideas where you would put it @bokobza ?